expression/aggregation/base_func.go

```diff
@@ -286,7 +286,7 @@ func (a *baseFuncDesc) typeInfer4MaxMin(ctx sessionctx.Context) {
 		a.Args[0] = expression.BuildCastFunction(ctx, a.Args[0], tp)
 	}
 	a.RetTp = a.Args[0].GetType()
-	if (a.Name == ast.AggFuncMax || a.Name == ast.AggFuncMin) && a.RetTp.GetType() != mysql.TypeBit {
+	if (a.Name == ast.AggFuncMax || a.Name == ast.AggFuncMin || a.Name == ast.AggFuncFirstRow) && a.RetTp.GetType() != mysql.TypeBit {
```

Comment:
Just wondering why the `TypeBit` is excluded?

Reply:
After a second thought, I think TypeBit should not be excluded.